cluster/src/main/java/io/scalecube/cluster/membership/MembershipProtocolImpl.java

```diff
@@ -94,13 +94,14 @@
    * @param scheduler scheduler
    */
   public MembershipProtocolImpl(
-      Member localMember,
-      Transport transport,
-      FailureDetector failureDetector,
-      GossipProtocol gossipProtocol,
-      MetadataStore metadataStore,
-      MembershipConfig config,
-      Scheduler scheduler) {
+    Member localMember,
+    Transport transport,
+    FailureDetector failureDetector,
+    GossipProtocol gossipProtocol,
+    MetadataStore metadataStore,
+    MembershipConfig config,
+    Scheduler scheduler) {
+
```

Comment:
Plz make sure u are using google-code-formatter. Reformat changed files with it.